src/toil/server/wes/toil_backend.py

```diff
@@ -63,15 +63,16 @@
 logging.basicConfig(level=logging.INFO)
 
 class ToilWorkflow:
-    def __init__(self, base_scratch_dir: str, state_store_url: str, run_id: str):
+    def __init__(self, base_work_dir: str, state_store_url: str, run_id: str):
         """
         Class to represent a Toil workflow. This class is responsible for
         launching workflow runs via Celery and retrieving data generated from
         them.
 
-        :param base_scratch_dir: The directory where workflows keep their
+        :param base_work_dir: The directory where workflows keep their
                                  output/scratch directories under their run
-                                 IDs.
+                                 IDs. Includes the MessageBus which requires
```

Comment:
I think the file is really a log of the message bus's messages, rather than being the bus *itself*, like a socket that lets you talk top it or something.